### DISCUSSION TITLE
Can't figure out how to stop AdGuardHome after using auto-install

### DISCUSSION THREAD

**Discussion by N0iSEA** [AdguardTeam/AdGuardHome#5541] [Q&A] [answered]
I used the the automated install at: [https://github.com/AdguardTeam/AdGuardHome#automated-install-linux-and-mac](url) to install on ubunutu 22.04. Everything has been working fine except that I haven't been able to figure out how to stop it so that i can get an SSL certificate. I have tried every command I could find on google (usually the guidance says `AdGuardHome -s stop' ) but nothing I have found works. 

When I do  'service  --status-all' there's no AdGuardHome service. Please help me. There is clearly something that I don't understand and need to learn but after 3 days of searching I have run out of ideas...

**Comment by ammnt** [comment]
@N0iSEA, try this:
`sudo /opt/AdGuardHome/AdGuardHome -s stop`

**Reply by N0iSEA** [reply]
Unfortunately that did not work either. I tried as many variations of that as I could think of.
thats is the file location but it returns 

sudo /opt/AdGuardHome/AdGuardHome: command not found

**Reply by ppfeufer** [reply]
Try:
```shell
sudo systemctl stop AdGuardHome.service
```

**Comment by N0iSEA** [comment]
It says Failed to stop AdguardHome.service: Unit AdguardHome.service is not
loaded.

It is true that I do not see AdguardHome service running when i list
running services. However, i have been using and am still able to access
adguard home in the web gui for the ip address of this server. How is this
possible?

I used the automatic installation from the link in my original post. I am
starting to think I should start over and build it myself so i can make
sure I can stop it and get the SSL.

The only reason that I hesitate to do that is that I have already it
configured to work with cloudflared and unbound and everything else about
it is working as intended.

**Reply by ppfeufer** [reply]
Does it say:
```
Failed to stop AdguardHome.service: Unit AdguardHome.service is not
```

or:
```
Failed to stop AdGuardHome.service: Unit AdGuardHome.service is not
```

Make sure to write the command exactly like I wrote it here. It's case-sensitive. (Feel free to provide screenshots from error messages, might make finding spelling errors easier.)

Some other things you can do to figure out where AGH is installed on your system since the command provided by @ammnt didn't work either is:
```shell
ps -aA | grep AdGuardHome
```

It should show something like this:
> **22426**       ?        00:07:19 AdGuardHome

The first number is the process ID, write it down.

Now run (replace `22426` with the process ID you got from the former command output):
```shell
sudo ls -l /proc/22426/exe
```

This should result in something like this
> lrwxrwxrwx 1 root root 0 Feb 27 12:05 /proc/22426/exe -> **/opt/AdGuardHome/AdGuardHome**

Now you have the path to your AGH installation and you can try again with:
```shell
sudo /opt/AdGuardHome/AdGuardHome -s stop
```
(replace `/opt/AdGuardHome/AdGuardHome` with the path you got from the former command)

**Comment by N0iSEA** [comment, marked as answer]
You were right. It was the capital G. Thank you. I am sorry for making such
a dumb mistake and wasting people’s time. I must have checked the case
sensitivity dozens of times but lizard brain and my eyes only focused on
the A and the H.

**Reply by ppfeufer** [reply]
If this solves your question, please feel free to mark this thread as answered.
